components/feature/downloads/src/main/java/mozilla/components/feature/downloads/DownloadNotification.kt

```diff
@@ -110,7 +110,7 @@ internal object DownloadNotification {
             val channel = NotificationChannel(
                 NOTIFICATION_CHANNEL_ID,
                 context.getString(R.string.mozac_feature_downloads_notification_channel),
-                NotificationManager.IMPORTANCE_DEFAULT
+                NotificationManager.IMPORTANCE_LOW
```

Comment:
Note that you cannot retroactively change the importance of a channel. If we'd really want to make this change for all existing users then we'd have to create a new channel id and remove the old channel.

Reply:
Couldn't we just remove the sound for the channel? (`.setSound(null,null)`)
This way we avoid creating a new channel, and also avoid issues with a lower importance (notification on channel with `IMPORTANCE_LOW` will not appear if the status bar already is filled with higher importance notifications.)

Reply:
I think in this case let's create a new channel ID. I'm fine with the low importance for download notifications.

Reply:
FYI: There are some reports that in Android Q even the `IMPORTANCE_LOW` priority plays a sound. I haven't tested that, though...

Reply:
Thanks all for the inputs. Updated - mostly used https://github.com/mozilla-mobile/android-components/commit/31a0c2fb6b7e891fd42a40f2d4285e31408521e4 as a reference.